<!-- -->

<launch>

	<node pkg="mpu6050_driver" type="mpu6050_driver_node" name="mpu6050_drive_node"  output="screen">
		<param name="port" value="/dev/ttyUSB1"/>
		<param name="baud_rate" value="115200"/>
		<param name="tf_parent_frame_id" value="imu_base"/>
		<param name="tf_frame_id" value="imu"/>
	</node>

</launch>
